Project/feature-u-soundboard-api/src/index.ts

```diff
@@ -28,6 +26,17 @@ app.use(cors(options));
 app.use(express.json());
 app.use(express.urlencoded());
 
+app.post("/tts", authenticateJWT, async (req, res) => {
+    try {
+        const file = await soundController.createTTS(req.body.text);
+        await soundController.save(req.body.text, file, req.user.email);
+        return res.json({
+            message: 'TTS created.'
+        })
```

Comment:
Is een status code 200 niet genoeg? of is er een specifieke rede woorom je een message terugstuurt?